src/models/multiBarHorizontalChart.js

```diff
@@ -24,6 +24,7 @@ nv.models.multiBarHorizontalChart = function() {
         , showXAxis = true
         , showYAxis = true
         , stacked = false
+        , reduceXTicks = true // if false a tick will show for every data point
```

Comment:
Unsubscribe


[novus_logo_email_signature]

Stanley Altshuller | Co-Founder, Chief Research Officer
Novus Partners Inc | 200 Park Avenue, 27th Floor  | New York, NY 10166
212.586.3032 | stan@novus.com<mailto:stan@novus.com>

From: liquidpele [mailto:notifications@github.com]
Sent: Tuesday, April 26, 2016 7:07 PM
To: novus/nvd3 <nvd3@noreply.github.com>
Subject: Re: [novus/nvd3] support for reduceXTicks in multiBarHorizontalChart (#1538)


In src/models/multiBarHorizontalChart.js<https://github.com/novus/nvd3/pull/1538#discussion_r61179886>:

> @@ -24,6 +24,7 @@ nv.models.multiBarHorizontalChart = function() {

>          , showXAxis = true

>          , showYAxis = true

>          , stacked = false

> +        , reduceXTicks = true // if false a tick will show for every data point

For new options, please also document them in the big json object in examples/documentation.html

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/novus/nvd3/pull/1538/files/51de81d745f626307faac557b03e689b452e8750#r61179886>

This e-mail message, and any attachments, is intended only for the use of the individual or entity identified in the alias address of this message and may contain information that is confidential, privileged and subject to legal restrictions and penalties regarding its unauthorized disclosure and use. Any unauthorized review, copying, disclosure, use or distribution is strictly prohibited. If you have received this e-mail message in error, please notify the sender immediately by reply e-mail and delete this message, and any attachments, from your system. Thank you.